src/client/Microsoft.Identity.Client/Internal/Logger/IdentityLoggerAdapter.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using Microsoft.Identity.Client.Core;
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.Internal.Logger
+{
+#if !XAMARINMAC20
```

Comment:
I dislike this very much. Please create a work item to explore getting rid of XamarinMac20